packages/mrc-ide-covidsim/src/params-serialization.ts

```diff
@@ -0,0 +1,98 @@
+import { DateTime } from 'luxon'
```

Comment:
Are there any actual changes in this file? Or did you just extract the existing text to a new file?